systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -235,23 +233,22 @@ void testSwitchingConnectStabilityIdentifiesFeatureGateOnAndOff(ExtensionContext
         final LabelSelector connectLabelSelector = KafkaConnectResource.getLabelSelector(testStorage.getClusterName(), KafkaConnectResources.deploymentName(testStorage.getClusterName()));
         Map<String, String> connectPods = PodUtils.podSnapshot(clusterOperator.getDeploymentNamespace(), connectLabelSelector);
 
+        String startingConnectorPodName = kubeClient().listPods(testStorage.getNamespaceName(), connectLabelSelector).get(0).getMetadata().getName();
+
         KafkaClients clients = new KafkaClientsBuilder()
-            .withProducerName(testStorage.getProducerName())
+            .withProducerName("starting-fg-producer")
```

Comment:
I think that you can have it in the variable and check also success of the client, for the previous client it wasn't done, but I guess it would be nice to have 🤷 

Then you can just update it with a new name.